tests/swift/Wasm.tests/Tests/FlatBuffers.Test.Swift.WasmTests/monster_test_generated.swift

```diff
@@ -5,29 +5,31 @@
 import FlatBuffers
 
 ///  Composite components of Monster color.
-public enum MyGame_Example_Color: UInt8, Enum, Verifiable {
+public struct MyGame_Example_Color: OptionSet, Enum, Verifiable {
   public typealias T = UInt8
+  public let rawValue: T
+
+  public init(rawValue: T) {
+    self.rawValue = rawValue
+  }
+
   public static var byteSize: Int { return MemoryLayout<UInt8>.size }
   public var value: UInt8 { return self.rawValue }
-  case red = 1
+  public static let red = MyGame_Example_Color(rawValue: 1)
   ///  \brief color Green
   ///  Green is bit_flag with value (1u << 1)
-  case green = 2
+  public static let green = MyGame_Example_Color(rawValue: 2)
   ///  \brief color Blue (1u << 3)
-  case blue = 8
+  public static let blue = MyGame_Example_Color(rawValue: 8)
 
-  public static var max: MyGame_Example_Color { return .blue }
-  public static var min: MyGame_Example_Color { return .red }
```

Comment:
I just noticed this today, let's keep the min/max as API and add the .none and .all for struct APIs only